backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/services/DetailsViewService.java

```diff
@@ -418,16 +417,11 @@ public boolean setTransitionSourceParameter(TransitionUsage transitionUsage, Ele
             succession.getFeatureMembership().stream()
                     .filter(EndFeatureMembership.class::isInstance)
                     .map(EndFeatureMembership.class::cast)
-                    .findFirst()
```

Comment:
Thank you for trying to improve things here, I appreciate.
Unfortunately, I'm not sure your version is better than the one existing.

Reply:
Done